### PR TITLE
chore: [VRD-726] Unhide client function

### PR DESCRIPTION
**Pull request by ghost**
## Impact and Context
Now that the feature is ready to go, make it not private anymore!

## Testing
- [X] Unit test
- [ ] Deployed to dev env
- [ ] Other (explain) 

Tests run yay!
## Reverting
- [ ] Contains Migration - _Do Not Revert_

### CODE REVIEW

File: client/verta/tests/unit_tests/.hypothesis/unicode_data/13.0.0/charmap.json.gz
Comment (liuverta) [inline client/verta/tests/unit_tests/.hypothesis/unicode_data/13.0.0/charmap.json.gz:1]: What dis

**Review by liuverta — CHANGES_REQUESTED** [review]

File: client/verta/tests/unit_tests/.hypothesis/unicode_data/13.0.0/charmap.json.gz
Comment (liuverta) [reply client/verta/tests/unit_tests/.hypothesis/unicode_data/13.0.0/charmap.json.gz:1]: Oh, I have `*.gz` in my `~/.gitignore_global`. That's why I've never seen this!

We might need to add `.hypothesis` to [the repo `.gitignore`](https://github.com/VertaAI/modeldb/blob/main/.gitignore).

Comment (ghost) [reply client/verta/tests/unit_tests/.hypothesis/unicode_data/13.0.0/charmap.json.gz:1]: darnit

File: client/.gitignore [hunk 1]
```diff
@@ -20,6 +20,7 @@ verta/verta.egg-info/
 
 # unit testing
 verta/tests/.hypothesis
+.hypothesis
```
Comment (ghost) [inline client/.gitignore:23]: generated new test results and made sure they didn't show up according to git!

**Review by liuverta — APPROVED** [review]

**Pull request merged**